.github/workflows/testWindows.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         timeout-minutes: 60      
         run: |
           cd "${{ inputs.unreal-binaries-path }}"
-          ./UnrealEditor-Cmd.exe "$env:TESTS_PROJECT_ROOT/TestsProject.uproject" -execcmds="Automation RunTests Cesium.;quit" -nullrhi -unattended -nosplash -ReportExportPath="$env:TESTS_PROJECT_LOGS"
```

Comment:
Move existing tests to Cesium.Unit.X namespace.
The new performance tests are Cesium.Performance.X, and I don't want to run these under CI